rfc/model_evaluation/rfc.md

```diff
@@ -0,0 +1,49 @@
+|     task                   |   String        |   Options: [classification, regression, forecasting]  Required: True  Description: Task type for which the model is trained                                                                                                                                                                                                                                                                                                                                                                       |
+|     model_uri              |   String        |   Required: False (Required only if mode is score or predict)  Description: MLFlow model uri (could be of type) ->        To Fetch models from an azureml run        - runs:/<azureml_run_id>/run-relative/path/to/model        To Fetch model from azureml model registry        - models:/<model_name>/<model_version>        - models:/<model_name>/<stage>  NOTE: model_uri has been added to accommodate current ML Designer as it doesn’t allow users to drag-and-drop Registered models    |
+|     mlflow_model           |   MLFlow Model  |   Required: False (Required only if mode is score or predict)  Description: MLFlow model (either registered or output of another job)                                                                                                                                                                                                                                                                                                                                                             |
+|     data_folder            |   URI Folder    |   Required: True  Description: Input folder which contains the following files ->        - Test Data (in csv format)        - Additional Parameters file (in JSON format)        - Any other artifact required for model evaluation like y_transformer (in pickled format)                                                                                                                                                                                                                        |
+|     test_data_file_name    |   String        |   Required: True  Description: Full path of test data file in above folder                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Why would a data folder contain an mlflow_model ? I'm not saying it could not, I understand why it might, but that's not according to best practices. And having folder + file name as distinct parameter is such a UX downer because if you change your file structure you have to change your params everywhere.

I'd suggest wondering from a UX perspective what's best here.

Reply:
After considering some of the Users' feedback and thorough UX team's review, we have made changes to the input parameters that we accepting in our Component. Ankush has updated the YAML and Designer view in RFC docs for reference